django_project/base/views/project.py

```diff
@@ -236,3 +241,102 @@ def get_redirect_url(self, slug):
         project.approved = True
         project.save()
         return reverse(self.pattern_name)
+
+
+def paymentview(request):
+    '''
+    :param request:
+    :return:
+    To do
+    1. Try catch blocks
+    2. Link to javascript form
+    3. Add webhooks
+    '''
+    stripe.api_key = "sk_test_OHW7bvLJhDtm1k2pI8AwIiEY"
+    ourclientid = "ca_BM5OIMXOFqSOBjJcr4DrGHTsK3tLFuW3"
+    myaccount = stripe.Account.retrieve("acct_1AzNDQGz66mVbJVl")
+    myaccountid = myaccount.id
+    '''
+        Stripe connect
+    '''
+    returnid = request.GET.get(
+        'code')  # token that is sent to retrieve client id to be stored in db
+    # POST request to OAUTH
+
+    r = requests.post(
+        "https://connect.stripe.com/oauth/token",
+        data={
+            'client_secret': stripe.api_key,
+            'code': returnid,
+            'grant_type': "authorization_code"})
+    temp = json.loads(r.content)
+    if r.status_code == 200:
+        merchantid = temp['stripe_user_id']
+    # Hard coded clientid returned- acct_1B1rtDBWHazZ6NrT
+    else:
+        merchantid = "acct_1B1rtDBWHazZ6NrT"
```

Comment:
let's put this into settings and import it from there

Reply:
Yeah isn't this dangerous? Hopefully the above id is just a test account ....